src/pages/dashboard.vue

```diff
@@ -170,12 +170,12 @@ const loadBusinessInfo = async (force = false) => {
       }
     } else {
       await business.loadBusiness(identifier, force)
+      await useBcrosTodos().loadTasks(identifier, true)
```

Comment:
This will make loading slower because this call is no longer done concurrently with the others, which is undesirable.

Why does this work? (I noticed that loadTasks() awaits functions that await other functions.)

Theoretically, the previous code should work. I'm trying to understand why it doesn't.

Reply:
This conversation is outdated. Resolving it.